src/_partials/ui/helpSupport.ejs

```diff
@@ -1,13 +1,10 @@
-<div class="medium-2 columns medium-offset-1">
+<div class="medium-2 columns medium-offset-2">
 	<a href="<%= url.ext.mailingList %>" class="icon mailingList">Dojo Mailing List</a>
 </div>
 <div class="medium-2 columns">
 	<a href="<%= url.ext.irc %>" class="icon irc">#dojo on IRC</a>
 </div>
 <div class="medium-2 columns">
-	<a href="<%= url.ext.gitter %>" class="icon irc">#dojo2 on gitter</a>
```

Comment:
We do have dojo 1 on gitter now, might be worth linking to that, as well as discourse at some point... that said, not a blocker